### PR TITLE
[CUB-4771] Use active user session in workflows

### PR DESCRIPTION
**Pull request by Thrashattack**
### Description

Add a new interceptor specification to the schema. 
WIll be used to identify when to inject the current user auth token in the webhook request header

### Manual PR Checklist

- [x] CHANGELOG entry added
- [x] Test coverage added
- [x] Licenses checked
- [x] All related Jira tickets added to PR title
- [x] Description in Jira ticket is clear and up-to-date

[CUB-4771]: https://labforward.atlassian.net/browse/CUB-4771?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/schemata/definitions/workflow-template/action-objects/webhook.yml [hunk 1]
```diff
@@ -33,6 +33,9 @@ additionalProperties:
                     type: string
                   access_key:
                     type: string
+              oidc:
+                type: object
+                additionalProperties: false
```
Comment (kgrhartlage) [inline src/schemata/definitions/workflow-template/action-objects/webhook.yml:38]: this will only allow an empty object as a value, is that the intention?

Comment (ignatiusreza) [reply src/schemata/definitions/workflow-template/action-objects/webhook.yml:38]: lets go with the following:

```
oneOf:
  - type: object
    additionalProperties: false
    properties:
      s3:
        type: object
        additionalProperties: false
        properties:
          secret_key:
            type: string
          access_key:
            type: string
  - type: string
    enum:
      - oidc
```

Comment (kgrhartlage) [reply src/schemata/definitions/workflow-template/action-objects/webhook.yml:38]:
```diff
-    enum:
-      - oidc
+ const: oidc
```

Comment (kgrhartlage) [reply src/schemata/definitions/workflow-template/action-objects/webhook.yml:38]: Can someone also write up a description for all this and explain what is what, what it does, possibly add external links if appropriate, etc. We should start with the basics and spell out the `oidc` acronym, explain the use case, etc.

Comment (ignatiusreza) [reply src/schemata/definitions/workflow-template/action-objects/webhook.yml:38]: oidc = OpenID Connect.. honestly, both is equally cryptic to someone who don't already know what it is.. :sweat_smile:

Comment (kgrhartlage) [reply src/schemata/definitions/workflow-template/action-objects/webhook.yml:38]: I know what it means. My point is that there is ZERO context here. Are you suggesting that it's a reasonable expectation for all workflow authors to look at these TWO LINES and be like _"Ah, of course, this is how I use OpenID Connect to inject the active user session in workflows."_

**Comment by Thrashattack** [comment]
@kgrhartlage If you run yarn build on this branch, it will remove some of the code in the dist files coming from the main branch. 
Is it correct?

**Comment by kgrhartlage** [comment]
Can you double check on your end, you might have had local changes when building. Failing tests also seem to suggest that

**Review by kgrhartlage — APPROVED** [review]
36j16g

**Pull request merged**